phpcs.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Coding Standards for Liveblog">
```

Comment:
Name should be `"Custom ruleset for Liveblog"`. It's a ruleset, not a standard.

Reply:
Also copied from amp-wp.

Reply:
This is just about being accurate and educating developers.

For your reference (since I see you on many different repos doing PRs improving PHPCS setups like this - and thank you for that!), here's my [plugin boilerplate `.phpcs.xml.dist`](https://github.com/GaryJones/plugin-boilerplate/blob/master/.phpcs.xml.dist) file, which applies many good practices.

Reply:
Fixed with 14a4a9e.